Osiris/Hacks/Esp.cpp

```diff
@@ -221,6 +221,32 @@ static void renderPlayerBox(Entity* entity, const Config::Esp::Player& config) n
             }
         }
 
+		if (config.activeWeapon)
+		{
+			static PlayerInfo playerInfo;
+			if (interfaces.engine->getPlayerInfo(entity->index(), playerInfo))
+			{
+				static wchar_t name[128];
+				if (MultiByteToWideChar(CP_UTF8, 0, entity->getActiveWeapon()->getWeaponData()->name, -1, name, 128))
```

Comment:
Use https://github.com/danielkrupinski/Osiris/blob/71c1846eb7bdceda82b66745b0795c9fc9f480c6/Osiris/SDK/Localize.h#L7

Reply:
I use the Localize find token and its working!!! Thank you very much for the hint!